CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+### 0.18.0
+
+This release expands support for interacting with iFrames using the css
+selector: `query("webView css:'iframe' css:'h1'")`.
```

Comment:
'tis generally true, assuming your `webVie` is visible and the `iframe` is visible.